src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -37,6 +38,8 @@ public class RobotContainer {
     // Dashboard inputs
     public final LoggedDashboardChooser<Command> autoChooser;
 
+    public final PhoenixTimeSyncSignalRefresher timeSyncSignalRefresher = new PhoenixTimeSyncSignalRefresher(SwerveConfig.CAN_BUS);
```

Comment:
if it's only used for odometry, call it `odometrySignalRefresher`

Reply:
The entire class, or just the field?

Reply:
The field